packages/cli/package.json

```diff
@@ -57,7 +57,7 @@
     "@graphql-tools/utils": "^10.0.1",
     "ajv": "^8.12.0",
     "change-case": "^4.1.2",
-    "cosmiconfig": "^9.0.0",
+    "cosmiconfig": "patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch",
```

Comment:
I think this is by mistake?

Reply:
Hummm that's a bit weird, not sure how it does work in this case, will check

Reply:
Ok so yes this is a applied globally.

The thing I don't understand is why before, we had patch for version 7 while having a forced resolution to v9

Reply:
Ok, I've reverted this and the patched version is now not applied to our own packages, which are using the actual v9 of cosmiconfig:

```
├─ @graphql-mesh/cli@workspace:packages/cli [959e1]
│  └─ cosmiconfig@npm:9.0.0 [a85b4] (via npm:^9.0.0 [a85b4])
│
├─ @graphql-mesh/cli@workspace:packages/cli [e94ec]
│  └─ cosmiconfig@npm:9.0.0 [a85b4] (via npm:^9.0.0 [a85b4])
│
├─ @graphql-mesh/cli@workspace:packages/cli
│  └─ cosmiconfig@npm:9.0.0 [a85b4] (via npm:^9.0.0 [a85b4])
│
├─ @svgr/core@npm:5.5.0
│  └─ cosmiconfig@patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch::version=7.1.0&hash=d5afe4 (via patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch)
│
├─ @svgr/plugin-svgo@npm:5.5.0
│  └─ cosmiconfig@patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch::version=7.1.0&hash=d5afe4 (via patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch)
│
├─ babel-plugin-macros@npm:3.1.0
│  └─ cosmiconfig@patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch::version=7.1.0&hash=d5afe4 (via patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch)
│
├─ fork-ts-checker-webpack-plugin@npm:6.5.3
│  └─ cosmiconfig@patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch::version=7.1.0&hash=d5afe4 (via patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch)
│
├─ fork-ts-checker-webpack-plugin@npm:6.5.3 [eddd9]
│  └─ cosmiconfig@patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch::version=7.1.0&hash=d5afe4 (via patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch)
│
├─ postcss-loader@npm:6.2.1
│  └─ cosmiconfig@patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch::version=7.1.0&hash=d5afe4 (via patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch)
│
└─ postcss-loader@npm:6.2.1 [4ba9f]
   └─ cosmiconfig@patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch::version=7.1.0&hash=d5afe4 (via patch:cosmiconfig@npm%3A7.1.0#~/.yarn/patches/cosmiconfig-npm-7.1.0-13a5090bcd.patch)```